src/bosh_azure_cpi/lib/cloud/azure/utils/helpers.rb

```diff
@@ -571,20 +575,20 @@ def flock(lock_name, mode)
     end
 
     def get_storage_account_type_by_instance_type(instance_type)
+      support_premium_storage?(instance_type) ? STORAGE_ACCOUNT_TYPE_PREMIUM_LRS : STORAGE_ACCOUNT_TYPE_STANDARD_LRS
+    end
+
+    def support_premium_storage?(instance_type)
```

Comment:
I think there should be some rest api to check whether the premium storage is supported?
if yes, even we do not do it in this change, we should file one issue to track it.

Reply:
Filed an issue